x-pack/solutions/observability/plugins/infra/common/alerting/logs/log_threshold/types.ts

```diff
@@ -7,10 +7,38 @@
 
 import { i18n } from '@kbn/i18n';
 import * as rt from 'io-ts';
-import { persistedLogViewReferenceRT } from '@kbn/logs-shared-plugin/common';
+import {
+  Comparator,
+  criterionRT,
+  RequiredRuleParamsRT,
+  OptionalRuleParamsRT,
+  countCriteriaRT,
+  ratioCriteriaRT,
+  timeUnitRT,
+  timeSizeRT,
+  groupByRT,
+} from '@kbn/response-ops-rule-params/log_threshold';
+import type {
+  LogThresholdParams,
+  TimeUnit,
+  RatioRuleParams,
+  CountRuleParams,
+} from '@kbn/response-ops-rule-params/log_threshold';
 import { commonSearchSuccessResponseFieldsRT } from '../../../utils/elasticsearch_runtime_types';
 
 export { LOG_THRESHOLD_ALERT_TYPE_ID as LOG_DOCUMENT_COUNT_RULE_TYPE_ID } from '@kbn/rule-data-utils';
+export {
```

Comment:
I am not sure importing things here and then exporting them again is the right approach. The places that need them should import directly from `@kbn/response-ops-rule-params`